lib/beaker/host.rb

```diff
@@ -234,7 +234,11 @@ def exec command, options={}
         output_callback = nil
       else
         @logger.debug "\n#{log_prefix} #{Time.new.strftime('%H:%M:%S')}$ #{cmdline}"
-        output_callback = logger.method(:host_output)
+        if @options[:preserve_host_output]
```

Comment:
Let's call this color_host_output or some such - preserve_host_output makes it sound like you won't get any output without this being set.